deliverables/README.md

```diff
@@ -11,21 +14,45 @@ Sign Up for Cloud9 and create a basic Rails project.
 ###Intro to Ruby
 ####Required
 
-What is an "interpreted" language?<br>
-What is IRB?<br>
-What are Objects?<br>
-What are Methods?<br>
-What are Classes?<br>
-What are Blocks?<br>
-What is an Array?<br>
-What is an Iterator?<br>
-What are hashes?<br>
-What is a library?<br>
-What is a gem?
+What is an "interpreted" language?  
+It is a programming language for which most of its implementations execute instructions directly, without previously compiling a program into machine-language instructions. The interpreter executes the program directly, translating each statement into a sequence of one or more subroutines already compiled into machine code.<br>
+
+What is IRB?  
+Interactive Ruby Shell (IRB or irb) is a REPL for programming in the object-oriented scripting language Ruby. The abbreviation irb comes from the fact that the filename extension for Ruby is ".rb", although interactive Ruby files do not have an extension of ".irb".<br>
+
+What are Objects?  
+Most everything in Ruby is an Object. All objects have an identity; they can also hold state and manifest behaviour by responding to messages. <br>  
+
+What are Methods?  
+They are very similar to functions in any other programming language. Ruby methods are used to bundle one or more repeatable statements into a single unit. Method names should begin with a lowercase letter. If you begin a method name with an uppercase letter, Ruby might think that it is a constant and hence can parse the call incorrectly. <br>
+
+What are Classes?    
+A class is the blueprint from which individual objects are created. It is also an object-oriented term.<br>
+
+What are Blocks?  
+A block consists of chunks of code which You assign a name to, The code in the block is always enclosed within braces ({}).and they are always invoked from a function with the same name as that of the block. You invoke a block by using the yield statement.<br>
+
+What is an Array?  
+They are simple lists, they store their information in a sequence.<br>
+
+What is an Iterator?    
+Iterators are nothing but methods supported by collections, return all the elements of a collection, one after the other. <br>
+
+What are hashes?    
+A Hash is a collection of key-value pairs and the indexing is done via arbitrary keys of any object type, not an integer index.<br>
+
+What is a library?    
+The main place where libraries are hosted is RubyGems.org, providing Ruby libs as gems.<br>
+
+What is a gem?  
+A gem is a  a self-contained format That can be distributed by RubyGems.
 
 Do the challenges at http://tryruby.org/levels/1/challenges/0.
 
+My Code School Link:   https://www.codeschool.com/account?__utma=1.438278617.1423533714.1423607869.1423630333.3&__utmb=1.6.10.1423630333&__utmc=1&__utmx=-&__utmz=1.1423533714.1.1.utmcsr=(direct)|utmccn=(direct)|utmcmd=(none)&__utmv=-&__utmk=199311790
+
 Do the RubyMonk “Ruby Primer” course: http://rubymonk.com/learning/books/1
+My Ruby Account https://rubymonk.com/account
```

Comment:
This url isn't specific to your account.  I click this and it takes me to *my* account.